Jenkinsfile.groovy

```diff
@@ -274,9 +271,7 @@ pipeline {
           sh "rm -rf ./snyk"
 
           // Prepare veracode
-          sh "mkdir -p veracode"
-          sh "mkdir -p veracode/demo"
-          sh "mkdir -p veracode/react-vapor"
+          sh "mkdir -p veracode/demo veracode/react-vapor"
```

Comment:
the `sh "mkdir -p veracode"` is no longer needed?

Reply:
no, see flag `-p` in mkdir command ;)